bluebase_api/lib/bluebase_api/generators/app_generator.rb

```diff
@@ -33,6 +33,11 @@ def bluebase_api_customization
       invoke :setup_git_and_github
       invoke :setup_heroku_apps
       invoke :outro
+      invoke :customize_app_directory
+      invoke :customize_public_directory
+      invoke :customize_vendor_directory
+      invoke :customize_lib_directory
+      invoke :customize_tmp_directory
```

Comment:
Did you move these below everything else to denote that these were methods you changed? If so, do you think they should stay in this position? (Kinda nitpicky)